### PR TITLE
CMDCT-3138: Updated WP and SAR Homepage (Verbiage + Styling)

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Updating the WP and SAR homepage to match the latest verbiage, as well as adding a link for `User Guide and Help File` download. 

**NOTE:** The links themselves are not available yet, they will be updated as part of a follow-up ticket. 

### Related ticket(s)

CMDCT-3138

---
### How to test

Log in as a state user and verify that the updated verbiage and new links match the Figma mockups:

Screenshot 2024-01-04 at 11 49 43 AM

Screenshot 2024-01-04 at 11 49 49 AM

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/verbiage/pages/home.ts [hunk 1]
```diff
@@ -23,36 +23,30 @@ export default {
...
+        text: "The MFP Work Plan will be created and submitted initially and then amended semi-annually. CMS will review and approve all MFP Work Plans and proposed changes.<br/><br/>The MFP Work Plan deadlines are May 1st for inclusion in January through June reporting (Reporting Period 1) and November 1st for inclusion in July through December reporting (Reporting Period 2). Updates to existing state or territory-specific initiatives or the addition of new initiatives can occur during either of these periods, however, changes to transition benchmark projections should be made between July - November 1st to align with program budget requests.<br/><br/>CMS approved changes will be integrated into the MFP recipient's Semi-Annual Progress Report for the following reporting period.",
```
Comment (berryd) [inline services/ui-src/src/verbiage/pages/home.ts:33]: Superduper pedantic question because I'm an old geezer:  just one space after periods?

Comment (karla-vm) [reply services/ui-src/src/verbiage/pages/home.ts:33]: I don't think we have a specific standard for that, but I'm happy to take suggestions! Should we be adding more spaces after a period?

Comment (gmrabian) [reply services/ui-src/src/verbiage/pages/home.ts:33]: Lol I think we've been doing one space after periods everywhere

Comment (berryd) [reply services/ui-src/src/verbiage/pages/home.ts:33]: If that's the standard!  I'm just an old fudd that was taught to put two after breaks like periods, semicolons, and colons.

Comment (berryd) [reply services/ui-src/src/verbiage/pages/home.ts:33]: https://silicon-valley.fandom.com/wiki/Tabs_vs._spaces

**Review by gmrabian — APPROVED** [review]

**Review by berryd — APPROVED** [review]

**Pull request merged**